ros2cli/ros2cli/node/direct.py

```diff
@@ -35,7 +33,7 @@ def timer_callback():
 
         node_name_suffix = getattr(
             args, 'node_name_suffix', '_%d' % os.getpid())
-        self.node = rclpy.create_node(HIDDEN_NODE_PREFIX + 'ros2cli_node' + node_name_suffix)
+        self.node = rclpy.create_node(CLI_NODE_NAME_PREFIX + node_name_suffix)
```

Comment:
Wouldn't this remove this node's status as 'hidden'?

Reply:
Yes, that's the intention. Some CLI nodes are hidden while others are not, and I haven't found a good reason for this discrepancy. Moreover, ROS1 CLI nodes all start up as visible. Since this hiding is just a CLI-level feature (rendered irrelevant by supplying the `-a` flag) I don't feel strongly either way but I feel like it should be consistent (either all hidden or all visible).

Reply:
I feel all CLI tools used for debugging or status checking should be hidden by default and minimally intrusive, to avoid everyone else's workstations debugging the same platform/network from cluttering the shared data bus of everyone else, e.g. increasing the signal to noise ratio of parameters/topics/participant namespaces as more developers/cli-tools share the same network rather than solely from the number of robots. This is just my opinion from working on robotic competitions and  hackathons during crowded events such as RoboCup, or shared research labs.

What CLI do you currently find to be inconsistent in being hidden?
Perhaps is worth fixing them as well to be hidden?

Reply:
`ros2 topic echo` is hidden while `ros2 topic pub` is not. I can change all of them to be hidden, that's fine, as long as it's consistent :)

Reply:
Updated: https://github.com/ros2/ros2cli/pull/158/commits/5d2d22d1618f7224fe9a5438d9222fe31317ba13